src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
 // write your code here
+const population = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+const populationArr = [...population].map(amount =>
```

Comment:
map doesn't mutate initial array, it returns a new one

```suggestion
const populationArr = population.map(amount =>
```